src/App.js

```diff
@@ -18,7 +18,7 @@ import Logo from "./components/Logo";
 import SidePanel from "./components/SidePanel";
 import Timeline from "./components/Timeline";
 import getLogoSrc from "./models/logo";
-import Map from "./models/Map";
+import {Map} from "treetracker-web-map-core";
```

Comment:
@dadiorchen @iib0011 we need to retain the Map from the master branch right??

Reply:
I changed it